### PR TITLE
rendered md and rst

### PR DESCRIPTION
**Pull request by karmadolkar**
https://phabricator.wikimedia.org/T150098 for this task.

### CODE REVIEW

**Comment by karmadolkar** [comment]
@crookedstorm please review this at your leisure. :)

File: images/renderer/renderer.py [hunk 1]
```diff
@@ -28,9 +30,27 @@ def render_ipynb(full_path, format):
         html, res = exporter.from_file(file_handle)
     return Response(html, mimetype='text/html')
 
+def render_md(full_path, format):
+    """
+    Render a given .md file
+    """
+    md = markdown.markdownFromFile(filename)
```
Comment (aborrero) [inline images/renderer/renderer.py:37]: where is this "filename" var set?

File: images/renderer/renderer.py [hunk 2]
```diff
@@ -28,9 +30,27 @@ def render_ipynb(full_path, format):
...
+    rmd = md.convert()
+    return rmd
+
+def render_rst(full_path, format):
+    """
+    Render a given .rst file
+    """
+    docutils.core.publish_file(source_path=full_path, destination path="output.html", writer_name="html")
```
Comment (aborrero) [inline images/renderer/renderer.py:45]: typo: destination_path.

Moreover, is this writing to the disk? Would it be possible to access the resulting HTML directly from memory?

**Comment by crookedstorm** [comment]
Closing to due to inactivity and conflicts

**Pull request closed without merging**